Misc updates to the README (#5526)
chef/inspec: README.md

### Install it via rubygems.org

> When installing from source, gem dependencies may require ruby build tools to be installed. (A compiler-free variant is available with reduced functionality; use `inspec-core-bin` and `inspec-core`.)

Reviewer: ```suggestion
Installing the Chef InSpec package from source may require installing ruby build tools to manage gem dependencies. (A compiler-free variant is available with reduced functionality; use `inspec-core-bin` and `inspec-core`.)
```

Changes for active voice.
I'm not sure about word 'manage' and I'm open to suggestions. Handle? Build?

## Kudos

> Chef InSpec was originally created by Christoph Hartmann (@chris-rock) and Dominik Richter (@arlimus).

Reviewer 2: I think those references should be in the [] brackets.

> Chef InSpec was originally created by Christoph Hartmann (@chris-rock) and Dominik Richter (@arlimus).

Reviewer: ```suggestion
Chef InSpec was originally created by Christoph Hartmann and Dominik Richter.
```

Those are Slack handles, but Chris and Dominik have moved on to different companies.

Author: They are also their github IDs.
